proc/stack.go

```diff
@@ -16,25 +16,30 @@ func (thread *Thread) ReturnAddress() (uint64, error) {
 
 // Returns the stack trace for thread
 // Note that it doesn't include the current frame and the locations in the array are return addresses not call addresses
-func (thread *Thread) Stacktrace(depth int) ([]Location, error) {
+func (thread *Thread) Stacktrace(depth int) (*Location, []Location, error) {
```

Comment:
I'd much rather augment `Stacktrace` to return the current location in `[]Location` as opposed to returning current location separately.